starfish/test/test_overlap_utils.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import xarray as xr
+
+from starfish import IntensityTable
+from starfish.test import test_utils
+from starfish.types import Coordinates, Features
+from starfish.types._constants import OverlapStrategy
+from starfish.util.overlap_utils import (
+    Area,
+    find_overlaps_of_xarrays,
+    remove_area_of_xarray,
+    sel_area_of_xarray
+)
+
+
+def create_intenisty_table_with_coords(area: Area, n_spots=10):
```

Comment:
```suggestion
def create_intensity_table_with_coords(area: Area, n_spots: int=10) -> IntensityTable:
```

Reply:
I'm a bit concerned that this method only creates spots along a diagonal.  Are there any shortcomings from this approach?